src/pages/workspace/expensifyCard/WorkspaceExpensifyCardDetailsPage.tsx

```diff
@@ -44,6 +45,11 @@ type WorkspaceExpensifyCardDetailsPageProps = PlatformStackScreenProps<
 function WorkspaceExpensifyCardDetailsPage({route}: WorkspaceExpensifyCardDetailsPageProps) {
     const {policyID, cardID, backTo} = route.params;
     const workspaceAccountID = useWorkspaceAccountID(policyID);
+    const domainCardsID = useDomainCardsID(policyID);
+
+    // TODO: add logic for choosing between the domain and workspace feed when both available
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const cardsID = domainCardsID || workspaceAccountID;
```

Comment:
I notice we also have `cardID`. Are `cardsID` and `cardID` different?

Reply:
yes, cardID is the id for a particular card, cardsID is the part of the 'private_expensifyCardSettings_' or 'cards_' key saved as fundID in the object - should I rename it for fundID?
<img width="557" alt="Screenshot 2025-04-07 at 16 03 02" src="https://github.com/user-attachments/assets/cea7356d-946d-4367-bdd8-b862057f27b2" />

Reply:
Oh yeah, calling it `fundID` sounds good to me.

Reply:
ok, so I'll rename to fundID